recipes-kernel/linux/linux-raspberrypi-dev.bb

```diff
@@ -24,6 +24,8 @@ SRC_URI = " \
     git://git.yoctoproject.org/yocto-kernel-cache;type=kmeta;name=meta;branch=${LINUX_RPI_KMETA_BRANCH};destsuffix=${KMETA} \
     file://powersave.cfg \
     file://android-drivers.cfg \
+    file://imx708-overlay.dts;subdir=git/arch/${ARCH}/boot/dts/overlays \
```

Comment:
Interesting approach. @kraj does this have any downsides? I don't see any - maybe when using devtool?

Reply:
I think it might be better to have it as a patch, if it gets added in one of future kernels then we wont be overwriting it without knowing.

Reply:
That is a fair point. @Jessseee do you reckon you can update this with a patch?

Reply:
Yes, I changed the addition of the imx708 dt-overlay to a patch.